test/functional/database-schema/column-types/sqlite/column-types-sqlite.ts

```diff
@@ -128,14 +128,8 @@ describe("database schema > column types > sqlite", () => {
         table!.findColumnByName("datetime")!.type.should.be.equal("datetime");
         table!.findColumnByName("simpleArray")!.type.should.be.equal("text");
         table!.findColumnByName("simpleJson")!.type.should.be.equal("text");
-        table!.findColumnByName("simpleEnum")!.type.should.be.equal("simple-enum");
-        table!.findColumnByName("simpleEnum")!.enum![0].should.be.equal("A");
```

Comment:
can you tell why did you remove those checks for data equality?

Reply:
It isn't data equality that is checked here. Data equality is still tested [on line 118](https://github.com/typeorm/typeorm/pull/3700/files#diff-ce7c0509b75f61f6f160d566ef049087R118). These tests were written to ensure that `findByColumnName("simpleEnum")` returned the original definition of the `simple-enum`, (instead of the actual `nvarchar` used by Sqlite). This is the behavior I am proposing to fix. So these tests are no longer relevant.